pyamg/classical/classical.py

```diff
@@ -35,23 +36,25 @@ def ruge_stuben_solver(A,
         of the linear system.  Method-specific parameters may be passed in
         using a tuple, e.g. strength=('symmetric',{'theta' : 0.25 }). If
         strength=None, all nonzero entries of the matrix are considered strong.
-    CF : string
+    CF : {string} : default 'RS'
         Method used for coarse grid selection (C/F splitting)
         Supported methods are RS, PMIS, PMISc, CLJP, CLJPc, and CR.
-    presmoother : string or dict
+    interpolation : {string} : default 'standard'
```

Comment:
If we rename 
`standard` --> `std_classical` | `classical`
then we'll have to rename things here